packages/web-client/app/components/card-pay/create-merchant-workflow/merchant-customization/index.ts

```diff
@@ -0,0 +1,151 @@
+import { action } from '@ember/object';
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import Layer2Network from '@cardstack/web-client/services/layer2-network';
+import { inject as service } from '@ember/service';
+import { WorkflowCardComponentArgs } from '@cardstack/web-client/models/workflow/workflow-card';
+import { didCancel, restartableTask, timeout } from 'ember-concurrency';
+import { taskFor } from 'ember-concurrency-ts';
+import { mostReadable, random as randomColor } from '@ctrl/tinycolor';
```

Comment:
Selected this package because it has first-class Typescript support, and provides a random color implementation that picks "nice" colors, and also has a nice method for the most readable color.